client-js/app/scripts/registry.ts

```diff
@@ -9,182 +11,10 @@ export interface RegistryListing {
   vals: Array<string>;
 }
 
-export interface RegistryApi {
```

Comment:
The `RegistryApi` just defined the abstract interface, while `RegistryServiceJsonRpc` is an implementation of that interface. I'd like to keep that separation of interface and implementation.